scripts/run-targeted-tests.mjs

```diff
@@ -89,14 +90,14 @@ async function distributeBetweenPipelines(modifiedProjects) {
           !isHandsontableTouched &&
           modifiedProjects.indexOf(projectName) >= Math.floor(modifiedProjects.length / pipelineCount)
         ) {
-          await spawnProcess(`npm run in ${projectName} test`);
+          await spawnProcess(`npm run in ${projectName} test -- --if-present`);
```

Comment:
What about the situation where someone adds a new project to the workspace with a missing `test` script (but that script should be there). In that case, the `run-targeted-tests` script wouldn't notice that. I'd not add the `--if-present` argument here as it may hide important feedback. 

I think it's mandatory for all packages to have a test script. But, in the case when the script is not necessary I'd add the happy script something like: ``"test": "echo \"Error: no test specified\" && exit 0"``. WDYT?

Reply:
That makes sense, but I'm not sure about this:

> I think it's mandatory for all packages to have a test script.

@wojciechczerniak ?

Reply:
I sort of agree with @budnix. By requiring `test` command we force author to make the decision and deliberately add or disable the command. This does not mean that the project will have a mandatory test script. Would be cool to have tests for docs tho